src/Stryker.CLI/Stryker.CLI/Stryker.CLI.csproj

```diff
@@ -4,12 +4,22 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
-    <PackageId>dotnet-stryker</PackageId>
-    <Authors>Richard</Authors>
-    <Company>InfoSupport</Company>
```

Comment:
You removed this. Maybe we should keep it. Better for visibility.

Reply:
The package id and author were moved to below. The company attribute was removed becouse this isn't directly an Info Support package. Or would you say the Company attribute should stay?

Reply:
Yeah.. maybe. What do you think @simondel ?